### PR TITLE
[4.0] Fix tween_cheatsheet 404 in Tween.xml

### PR DESCRIPTION
**Pull request by sk757a**

### CODE REVIEW

**Comment by aaronfranke** [comment]
Instead of having this PR, we could cherry-pick the original PR: https://github.com/godotengine/godot/pull/77941

**Comment by AThousandShips** [comment]
This isn't right, the link should be to the PNG in the 4.0 version, the 4.1 version isn't applicable, as the transition types aren't available in 4.0

File: doc/classes/Tween.xml [hunk 1]
```diff
@@ -88,7 +88,7 @@
 		[/csharp]
 		[/codeblocks]
 		Some [Tweener]s use transitions and eases. The first accepts a [enum TransitionType] constant, and refers to the way the timing of the animation is handled (see [url=https://easings.net/]easings.net[/url] for some examples). The second accepts an [enum EaseType] constant, and controls where the [code]trans_type[/code] is applied to the interpolation (in the beginning, the end, or both). If you don't know which transition and easing to pick, you can try different [enum TransitionType] constants with [constant EASE_IN_OUT], and use the one that looks best.
-		[url=https://raw.githubusercontent.com/godotengine/godot-docs/master/img/tween_cheatsheet.png]Tween easing and transition types cheatsheet[/url]
+		[url=https://raw.githubusercontent.com/godotengine/godot-docs/master/img/tween_cheatsheet.webp]Tween easing and transition types cheatsheet[/url]
```
Comment (AThousandShips) [inline doc/classes/Tween.xml:91]:
```suggestion
		[url=https://raw.githubusercontent.com/godotengine/godot-docs/4.0/img/tween_cheatsheet.png]Tween easing and transition types cheatsheet[/url]
```

**Review by AThousandShips — CHANGES_REQUESTED** [review]
As per above comment

**Review by AThousandShips — APPROVED** [review]

**Comment by AThousandShips** [comment]
Please squash your commits into one, see [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase)

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Pull request merged**